dataduct/pipeline/redshift_node.py

```diff
@@ -16,7 +16,9 @@ def __init__(self,
                  schedule,
                  redshift_database,
                  schema_name,
-                 table_name):
+                 table_name,
+                 create_table_sql,
```

Comment:
can you have default values for them as these are not passed from steps like extract-redshift etc ?